extensions/sql-database-projects/src/tools/packageHelper.ts

```diff
@@ -72,7 +72,7 @@ export class PackageHelper {
 				});
 			}
 		} catch (e) {
-			void vscode.window.showErrorMessage(e.message);
+			void vscode.window.showErrorMessage(constants.addSqlBindingPackageError, constants.checkoutOutputMessage);
```

Comment:
does this need to wait for the result from `showErrorMessage()` to show the output channel? Like in https://github.com/microsoft/azuredatastudio/blob/b8ea493f8cca96d608a560834a2779ea9aae1f5d/extensions/sql-database-projects/src/common/utils.ts#L519-L522

Reply:
oh actually maybe you can reuse `showErrorMessageWithOutputChannel()` here

Reply:
ah yes thanks for catching that - added the fix